### PR TITLE
Fix licence link

### PR DESCRIPTION
**Pull request by mjenczmyk**
~~I've changed licence icon to display GPL licence type and redirect on click to `COPYING` file.~~

What was changed is setting proper licence in `setup.py`, so licence link button will be displayed according to its content.

### CODE REVIEW

**Comment by mmaslowskicc** [comment]
Can we fix it instead by listing the appropriate license in `setup.py` so `shields.io` can get this from PyPI?

**Comment by mjenczmyk** [comment]
I've done as you @mmaslowskicc suggested.

File: setup.py [hunk 1]
```diff
@@ -94,6 +95,7 @@ def read(fname):
         'Topic :: Software Development :: Quality Assurance',
         'Topic :: Software Development :: Testing',
         'Topic :: Utilities',
+        'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
```
Comment (mmaslowskicc) [inline setup.py]: `License :: OSI Approved :: GNU Lesser General Public License v3 or later (LGPLv3+)`

File: setup.py [hunk 2]
```diff
@@ -86,6 +86,7 @@ def read(fname):
     keywords='tests py.test pytest fixture redis mongo mongodb'
              ' rabbit rabbitmq mysql postgres postgresql'
              ' elasticsearch',
+    license="GNU Lesser GPL",
```
Comment (mmaslowskicc) [inline setup.py]: Something like `GNU LGPLv3+`? Version number and "or later" are important. I don't know how this field usually looks like.

**Comment by mmaslowskicc** [comment]
Looks ok.

**Pull request merged**